packages/core/admin/ee/server/services/audit-logs/process-event.js

```diff
@@ -13,6 +13,10 @@ module.exports = function processEvent(strapi, name, ...args) {
     'media.update': getDefaultPayload,
     'media.delete': getDefaultPayload,
     'admin.auth.success': getDefaultPayload,
+    'role.create': getDefaultPayload,
+    'role.update': getDefaultPayload,
+    'role.delete': getDefaultPayload,
+    'role.permissions': getDefaultPayload,
```

Comment:
This is creating a conflict, you'll have to merge the latest version of the feature branch and add these events to the defaultEvents array